imports/models/BoardCollection.ts

```diff
@@ -10,6 +10,9 @@ export type BoardRecord = IBoard & {
 }
 
 export class BoardCollectionClass extends BaseCollection<BoardRecord> {
+  searchBoards() {
```

Comment:
make it a public method, pass `keyword: string` as parameter. use

```typescript
db.collection.find({
  $or: [
    {
      "elements.text": {
        $regex: "hello",
        $options: "i"
      }
    },
    {
      "title": {
        $regex: "hello",
        $options: "i"
      }
    }
  ]
})
```

to filter boards which `title` or text elements contains search keywords.

Reply:
check out this example: https://mongoplayground.net/p/D4PPy8eg38A

to see how i query boards which text elements or title contains "hello" keyword

hope this helps